tilequeue/command.py

```diff
@@ -949,6 +953,41 @@ def tilequeue_enqueue_tiles_of_interest(cfg, peripherals):
     logger.info('%d tiles of interest processed' % n_toi)
 
 
+def tilequeue_consume_tile_traffic(cfg, peripherals):
+    logger = make_logger(cfg, 'consume_tile_traffic')
+    logger.info('Consuming tile traffic logs ...')
+    logger.info(cfg.tile_traffic_log_path)
+
+    iped_dated_coords = None
+    with open(cfg.tile_traffic_log_path, 'r') as log_file:
+        iped_dated_coords = parse_log_file(log_file)
+
+    if not iped_dated_coords:
+        logger.info("Couldn't parse log file")
+        sys.exit(1)
+    
+    conn_info = dict(cfg.postgresql_conn_info)
+    dbnames = conn_info.pop('dbnames')
+    sql_conn_pool = DBAffinityConnectionsNoLimit(dbnames, conn_info, False)
+    sql_conn = sql_conn_pool.get_conns(1)[0]
+    with sql_conn.cursor() as cursor:
+        mimic_prune_tiles_of_interest_sql_structure(cursor)
+        postgres_add_compat_date_utils(cursor)
+        
+        # insert the log records after the latest_date
+        cursor.execute('SELECT max(date) from tile_traffic_v4')
+        max_timestamp = cursor.fetchone()[0]
+        iped_dated_coords_to_insert = filter(lambda iped_dated_coord: iped_dated_coord[1] > max_timestamp, iped_dated_coords) if max_timestamp else iped_dated_coords
+        for (host, timestamp, marchalled_coord) in iped_dated_coords_to_insert:
+            coord = coord_unmarshall_int(marchalled_coord)
+            cursor.execute("INSERT into tile_traffic_v4 (date, z, x, y, tilesize, service, host) VALUES ('%s', %d, %d, %d, %d, '%s', '%s')"
+                            % (timestamp, coord.zoom, coord.column, coord.row, 512, 'vector-tiles', host))
+
+        logger.info('Inserted %d records' % len(iped_dated_coords_to_insert))
```

Comment:
addressed in https://github.com/tilezen/tilequeue/pull/204/commits/e95e234d89c5531f46582ff3833743e31703061b